haystack/schema.py

```diff
@@ -131,28 +140,31 @@ def _get_id(self, id_hash_keys: Optional[List[str]] = None):
 
         if final_hash_key == "":
             raise ValueError(
-                f"Cant't create 'Document': 'id_hash_keys' must contain at least one of ['content', 'meta']"
+                "Cant't create 'Document': 'id_hash_keys' must contain at least one of ['content', 'meta'] or be set to None."
             )
 
         return "{:02x}".format(mmh3.hash128(final_hash_key, signed=False))
 
-    def to_dict(self, field_map={}) -> Dict:
+    def to_dict(self, field_map: Optional[Dict[str, Any]] = None) -> Dict:
         """
         Convert Document to dict. An optional field_map can be supplied to change the names of the keys in the
         resulting dict. This way you can work with standardized Document objects in Haystack, but adjust the format that
         they are serialized / stored in other places (e.g. elasticsearch)
         Example:
 
-        ```python
-        doc = Document(content="some text", content_type="text")
-        doc.to_dict(field_map={"custom_content_field": "content"})
+        ```
+            doc = Document(content="some text", content_type="text")
+            doc.to_dict(field_map={"custom_content_field": "content"})
 
-        # Returns {"custom_content_field": "some text", "content_type": "text"}
+            >>> {"custom_content_field": "some text", content_type": "text"}
```

Comment:
Why did you remove `python` here? I think this improves readability of our docs, as it adds syntax highlighting (see [current version](https://docs.haystack.deepset.ai/reference/primitives-api#documentto_dict))

Reply:
A merge commit I suppose. Thanks for spotting!